suppkg_rebuild.py

```diff
@@ -151,7 +112,7 @@ def check_multiple_specs(self, project, packages):
 
     def crawl(self):
         """Main method"""
-        rebuild_data = self.load_rebuild_data(self.project + ':Staging', 'dashboard', 'support_pkg_rebuild')
+        rebuild_data = self.api.load_file_content(self.project + ':Staging', 'dashboard', 'support_pkg_rebuild')
```

Comment:
`.read()` from `api.load_file_content` will be plain string then the following `.parse()` won't do.